doc/api/Playback_Information/.docConfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "pages": [
+    {
+      "path": "./getVideoBufferGap.md",
+      "displayName": "getVideoBufferGap"
+    },
+    {
+      "path": "./isContentLoaded.md",
+      "displayName": "isContentLoaded"
+    },
+    {
+      "path": "./isBuffering.md",
+      "displayName": "isBuffering"
+    },{
+      "path": "./isPaused.md",
+      "displayName": "isPaused"
+    }
+  ]
```

Comment:
We could also merge it with the content of `Content_Information`, all of its methods would still make sense as `Playback_Information` and differentiating between the two seems hard to me.